app/services/publishers/dfe_sign_in/update_users_in_db.rb

```diff
@@ -0,0 +1,49 @@
+require "dfe_sign_in/api"
+
+module Publishers::DfeSignIn
+  class UpdateUsersInDb
+    include DfeSignIn::API
+    include Publishers::DfeSignIn::Parsing
```

Comment:
This class has nothing to do with BigQ exports. 
Both share the DSI API and Parsing code through composition while keeping the BigQ export-related code exclusively within its namespace/classes.